platform/view/services/kvs/kvs_test.go

```diff
@@ -125,9 +127,10 @@ func testRound(t *testing.T, cfg *fakeProv) {
 
 	for ctr := 0; it.HasNext(); ctr++ {
 		val = &stuff{}
-		_, err = it.Next(val)
+		key, err := it.Next(val)
 		assert.NoError(t, err)
 		if ctr == 0 {
+			assert.Equal(t, k1, key)
 			assert.Equal(t, &stuff{"santa", 1}, val)
 		} else {
 			assert.Fail(t, "expected 2 entries in the range, found more")
```

Comment:
do you want to take the chance to fix the error message here? It should be "expected 1 entry in the range, ..."